src/lineup/internal/column.ts

```diff
@@ -30,9 +30,12 @@ export function addLazyColumn(colDesc: any, data: Promise<IScoreRow<any>[]>, pro
   //mark as lazy loaded
   (<any>colDesc).lazyLoaded = true;
   const col = provider.create(colDesc);
-  if(position === undefined || position === null) {
+  if (position == null) {
     ranking.push(col);
   } else {
+    if (position < 0) {
+      position = ranking.children.length - -position + 1; // -- since negative
```

Comment:
yes it has the same effect and you can suggest a change. However, my initial way was to amplify that you usually wanna count from backwards (thus length - X) but since the position is already negative, you also negate it first (--).